### PR TITLE
Refactor/rectification swagger

### PR DESCRIPTION
**Pull request by juanmaMacGyverCode**
Refactored the Swagger and fixed the deficiencies.

### CODE REVIEW

File: backend/src/main/java/com/huellapositiva/application/dto/GetProfileResponseDto.java [hunk 1]
```diff
@@ -29,7 +31,7 @@
     private final String birthDate;
 
     @Schema(
-            description = "User's telephone. TIP: The international prefix can be +X, +XX, +XXX",
+            description = "User's telephone. TIP: The international preffix can be +X, +XX, +XXX",
```
Comment (ibaiul) [inline backend/src/main/java/com/huellapositiva/application/dto/GetProfileResponseDto.java]: prefix

File: backend/src/test/java/com/huellapositiva/api/VolunteerControllerShould.java [hunk 2]
```diff
@@ -5,6 +5,7 @@
 import com.huellapositiva.application.dto.GetProfileResponseDto;
 import com.huellapositiva.application.dto.JwtResponseDto;
 import com.huellapositiva.application.dto.UpdateProfileRequestDto;
+import com.huellapositiva.application.dto.GetProfileResponseDto;
```
Comment (ibaiul) [inline backend/src/test/java/com/huellapositiva/api/VolunteerControllerShould.java]: Antes no era necesario ese import?

Comment (ibaiul) [reply backend/src/test/java/com/huellapositiva/api/VolunteerControllerShould.java]: ProfileDtoDataEntry.java esta vacio

Comment (juanmaMacGyverCode) [reply backend/src/test/java/com/huellapositiva/api/VolunteerControllerShould.java]: Hecho

File: backend/src/main/java/com/huellapositiva/application/dto/GetProfileResponseDto.java (hunk 1 above)
Comment (juanmaMacGyverCode) [reply backend/src/main/java/com/huellapositiva/application/dto/GetProfileResponseDto.java]: Hecho

**Review by ibaiul — DISMISSED** [review]

**Pull request merged**